models/CreateNewPatientRequest.js

```diff
@@ -4,18 +4,19 @@ Patient {
 
   family : string
   given : string
-  suffix : string
-  phone : {
+  --middle : string
+  --suffix : string
+  phone : [{
     value : string
     use : enum (home, work, temp, old, mobile)
-  }
+  }]
   email : array of strings
   gender : enum (male, female, other, unknown)
   birthdate : string
   race : enum (American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Black or African-American, White, Other Race)
   ethnicity: enum (Hispanic, Not Hispanic)
-  permissionToFollowup: boolean
   address: {
+    use: enum (home, work, temp)
     line: string
     city: string
     state: string
```

Comment:
While we're here, could you extend the relationship enum to have some of the values from this list? https://www.hl7.org/fhir/valueset-relatedperson-relationshiptype.html

The ones we need to send to WIR are Brother, Caregiver, Child, Foster Child, Father, Guardian, Grandparent, Mother, Other, Parent, Stepchild, Self (not sure why this is in the list), Sibling, Sister, and Spouse.

Reply:
FYI - The options per FHIR that we currently have in the UI are: Child, Foster Child, Spouse, Domestic Partner, Parent, In-Law, Grandparent, Other Family Member.
We only have 2 discrepancies:  Domestic Partner and In-Law.  I think both of these are valid options, so could we keep them as options in FHIR, but interpret them as "Other" when sending to WIR?

Reply:
Oh - the "contact" data structure you have is for the emergency contact.  In that case, the options you provide in the enum match the options given by FHIR.

In addition to this, we have a link to "other," which is a link to a RelatedPerson resource.  The RelatedPerson resource is what I was envisioning would capture the more detailed relationship attribute (like Parent, Child, Spouse, etc.).  Please see https://www.hl7.org/fhir/patient.html and https://www.hl7.org/fhir/relatedperson.html

Reply:
I'm OK with having the extra values map to "Other", as long as most of the values map to something more specific.

Reply:
I've updated the contact relationship enum. What do you think about extending the options in the UI to contain those the WIR accepts?
About RelatedPerson, do we need this resource type? If it is just storing the relationship type, this could be done in contact.relationship, which is an extensible valueset. RelatedPerson seems to store a subset of the information contained in Patient, meaning either the information is redundant in a RelatedPerson resource, or if we dont populate those fields, I question whether the resource is necessary.

Reply:
I think we should discuss this when we discuss the observation endpoint.  We definitely could just use the contact.relationship; I'm not sure if there is added value in establishing bi-directional linking with RelatedPerson.  For example, suppose:

- Household member #1 has household member #2 as their emergency contact (contact.relationship)
- Household member #2 has household member #1 as their emergency contact (contact.relationship)
- Household member #3 has household member #1 as their emergency contact (contact.relationship)
- Household member #4 has household member #1 as their emergency contact (contact.relationship)
Is there any real benefit to having a series of RelatedPerson references in the household member #1 Patient resource to be able to quickly tell that this person is connected to household members #2 through #4?  If not, then I definitely agree with excluding RelatedPerson.